webpack/assets/javascripts/react_app/redux/reducers/index.js

```diff
@@ -1,21 +1,21 @@
 import { combineReducers } from 'redux';
-import { reducer as reduxFormReducer } from 'redux-form';
+import { reducer as form } from 'redux-form';
 import bookmarks from './bookmarks';
 import statistics from './statistics';
 import hosts from './hosts';
 import notifications from './notifications/';
 import toasts from './toasts';
-import passwordStrength from './user/passwordStrength';
+import { reducers as passwordStrengthReducers } from '../../components/PasswordStrength';
 
 export function combineReducersAsync(asyncReducers) {
   return combineReducers({
     bookmarks,
-    form: reduxFormReducer,
+    form,
     statistics,
     hosts,
     notifications,
     toasts,
-    passwordStrength,
+    ...passwordStrengthReducers,
```

Comment:
See:
https://github.com/theforeman/foreman/pull/5240/files#diff-be94fe9aa3f3ab8239d637d315cfb884R19

Reply:
it a bit confused situation when you have both ways implemented here.
IMHO if we adopt this approach (which i can see some benefits from) we should change this entire file

Reply:
Will do it in a different pr :+1: